core/tx_pool.go

```diff
@@ -860,7 +892,11 @@ func (pool *TxPool) AddLocals(txs []*types.Transaction) []error {
 // If the senders are not among the locally tracked ones, full pricing constraints
 // will apply.
 func (pool *TxPool) AddRemotes(txs []*types.Transaction) []error {
-	return pool.addTxs(txs, false)
+	errs := []error{}
+	for _, err := range pool.addTxs(txs, false) {
```

Comment:
why doing this?

Reply:
goimports test ran on Travis require that I use the built-in `errors` package for the unit tests and does not allow me to use `github.com/pkg/errors`